docs/modules/documentation/containers/action-bar/examples/action-bar-back-example.component.html

```diff
@@ -8,6 +8,6 @@
     </fd-action-bar-header>
     <fd-action-bar-actions>
         <button fd-button [fdType]="'primary'">Cancel</button>
-        <button fd-button [fdType]="'main.scss'">Save</button>
```

Comment:
This stuff is due to a weird merge error I stumbled upon while merging the big API documentation PR, its fixed as part of this PR since its small